Unity.gitignore

```diff
@@ -25,6 +25,7 @@
 
 # Visual Studio cache directory
 .vs/
+.vscode/
```

Comment:
It's not a cache directory. We have VS code settings and extensions recommendations commited and not ignored. I wonder what the recommandation is though.